control/subscription_group.go

```diff
@@ -383,7 +377,7 @@ func (s *subscriptionGroup) subscribePlugins(id string,
 	for i, sub := range plugins {
 		plg, err := s.pluginManager.get(key(sub))
 		if err != nil {
-			serrs = append(serrs, serror.New(err))
+			serrs = append(serrs, pluginNotFoundError(sub))
```

Comment:
PluginNotFound is not the only error that can be returned from the `pluginManager.get` call. Do we need to worry about badkey errors also?